test/new_relic/agent/serverless_handler_test.rb

```diff
@@ -423,6 +423,47 @@ def test_metadata_for_payload_v2
         refute_empty metadata[:agent_language]
       end
 
+      # when testing locally with (the Node.js tool) serverless or whatnot, the
```

Comment:
Is one of the Node tools called "whatnot"? Or are you referring to "whatnot" more generally? (my guess is the latter, but I could see myself combing the internet for a tool called "whatnot")

Reply:
Disambiguated with [c869cd9](https://github.com/newrelic/newrelic-ruby-agent/pull/2821/commits/c869cd920660ec65c5b77ab641c34f7ff7cbefc2)